config/v1/types_image.go

```diff
@@ -67,6 +68,14 @@ type ImageSpec struct {
 	// internal cluster registry.
 	// +optional
 	RegistrySources RegistrySources `json:"registrySources"`
+
+	// imageStreamImportMode controls the import mode behaviour of imagestreams.
+	// It can be set to `Legacy` or `PreserveOriginal`. If this value is specified,
```

Comment:
What do these two values do? What would an end user expect to see when they set one or the other?

Reply:
it sets the `importMode` field in an imagestream to control importing a single arch manifest vs the manifestlisted image: https://docs.openshift.com/container-platform/4.15/openshift_images/image-streams-manage.html#images-imagestream-specify-architecture_image-streams-managing

Reply:
Can you explain that in simple terms that don't need background knowledge, and then we can add that into the godoc? Something like `Legacy means ...`, `PreserveOriginal means ...`

Reply:
i believe this is covered by adding the explanation to the fields themselves.

Reply:
You need it in the fields, you've currently got it set only on the enum constants.